components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/GroupImpl.java

```diff
@@ -50,6 +50,9 @@
 @SuppressWarnings("NonJaxWsWebServices")
 public class GroupImpl implements Group {
 
+    private static final String GROUP_CANNOT_NULL_MSG = "Group cannot be null.";
+    private static final String GROUP_NAME_INVALID_MSG = "Provided group name is invalid. Should be in minimum 3 " +
```

Comment:
Check the syntax of the error message. Better to change it to "Should be at least 3 "